### ISSUE TITLE
1983 missing data

### ISSUE THREAD

**Issue by cblevins** [regan008/MGG-LC-Data#15]
It looks like nobody transcribed pages 188-189 (end of Wisconsin to Wyoming/Canada) in the 1983 issue.

**Comment by regan008** [comment]
@tmmarks can you transcribe these for us? Should go in the 1983 guide in Airtable.

**Comment by regan008** [comment]
@haileybrink would you be able to transcribe these missing cities for us in 1983? Can you also spot check and just make sure we dont have any other pages that are missing from the dataset? Thank you SO much!

**Comment by haileybrink** [comment]
Yes, I can do that! I will put it on my list for next week.

Thank you,
Hailey
________________________________

**Comment by regan008** [comment]
@haileybrink great. And then when you are finished transcribing those are ready to do data verification checks on this year, here's the updated checklist for this year: 

- [ ] States should be listed with abbreviations not full names. i.e. CA not California. Is an issue in at least parts of 85, 81, 77.
- [ ] Washington DC entries should be listed as Washington in city rather than Washington, D.C..  "DC" should be listed in the state column. 
- [ ] Some cities need to be more specific. Look through to find any with a / splitting metro areas. Check guides and if no more specific location is listed, use [this geocoding tool](https://developers-dot-devsite-v2-prod.appspot.com/maps/documentation/utils/geocoder/) to try and figure out which locality the address is in.
- [ ] verify locations with blank cities to ensure they are statewide
- [ ] Filter and check that locations without addresses are marked unclear. PO Boxes too.
- [ ] Check to ensure that Puerto Rico entries are transcribed and listed as PR in state column

**Comment by haileybrink** [comment]
Should the state be D.C. or DC? For 1975 and 1977 it is D.C., in 1983 it is DC - can change to be either just wondering if it matters. Thank you!

**Comment by regan008** [comment]
For ease of data crunching it should be DC!

**Comment by haileybrink** [comment]
Got it! Made those changes :)

**Comment by regan008** [comment]
@haileybrink Great, thank you! Does this mean 1983 is done? Have all the items on the checklist been completed? If so could you check those off so we can mark this as complete and download the file from airtable?

Thanks for your work on this!!

**Comment by haileybrink** [comment]
I haven't finished the checklist yet, but will do that when I work this week :) thank you!

**Comment by haileybrink** [comment]
Working on this now, should the Virgin Islands be spelled out or abbreviated? If so, is the abbreviation VI?

**Comment by regan008** [comment]
Awesome - yes let's do VI for Virgin Islands. Thanks!

**Comment by haileybrink** [comment]
Okay, the checklist for 1983 is complete! I just had a few questions I will paste below. I am also good to go for more assignments! 

Questions/comments: 
- Should Miami Beach listings be changed from Miami to Miami Beach? 
- The listing that has the city as Hopland/Talmadge was not clarified when I used the geocoding link - I was unable to decipher where this address is - I left it as is but wanted to flag it for you as it is a remaining splitting metro area - in an advertisement for “The Ranch” on page 57, it is listed as in Ukiah California - so maybe it should be changed to that? 
- Ran into the same issue with statewide entries - some listed as statewide in the scanned issue are also listed as statewide in the Airtable, for some states however, the statewide entries in the scanned issue, have been listed under the cities in their address in the Airtable (example of this is Florida) - I know there is nothing I should do about this, just wanted to flag it here as well - basically, all of the blank cities in Airtable are statewide entries, but not all of the statewide entries in the scanned issue, have been recorded as statewide (city-less) in Airtable

**Comment by cblevins** [comment]
> * Should Miami Beach listings be changed from Miami to Miami Beach?
**Yes - good catch.**
> * The listing that has the city as Hopland/Talmadge was not clarified when I used the geocoding link - I was unable to decipher where this address is - I left it as is but wanted to flag it for you as it is a remaining splitting metro area - in an advertisement for “The Ranch” on page 57, it is listed as in Ukiah California - so maybe it should be changed to that?
**I took a quick Google and based on area code and town I think it's diff from Ukiah location. I would write city as Hopland, I think the geocode should work with this.** 
> * Ran into the same issue with statewide entries - some listed as statewide in the scanned issue are also listed as statewide in the Airtable, for some states however, the statewide entries in the scanned issue, have been listed under the cities in their address in the Airtable (example of this is Florida) - I know there is nothing I should do about this, just wanted to flag it here as well - basically, all of the blank cities in Airtable are statewide entries, but not all of the statewide entries in the scanned issue, have been recorded as statewide (city-less) in Airtable
**@regan008 can you double check this, but my memory is that for listings that appear under statewide in PDF we're still recording the address/city if one is listed, and including a note that says statewide somewhere? So if a note for statewide is not included in Airtable then I think we need to add one?**

**Comment by regan008** [comment]
@cblevins @haileybrink Yes, thats my memory of what we decided too re-statewide locations. If there is an address or city listed, we should use that. I just added a statewide.address checkbox where we can record the entries that we do this for. @lsmith13 is going through all the statewide locations and double checking this though, so don't worry too much about it.

**Comment by cblevins** [comment]
@haileybrink @lsmith13 Just wanted to check and see if either of you were still working on the final data cleaning for this 1983 guide, or whether we are good to go to start analyzing it. Thanks!

**Comment by regan008** [comment]
@haileybrink  Can you check to ensure NJ is complete in this year? @lsmith13 noticed an issue with those listings while working on #19.

**Comment by haileybrink** [comment]
Yes, I can do that! I was assigned 1983, but I think it was assigned to both me and lsmith13 so I did not start on it. I will start it when I work this week! Unless lsmith13 has already done it?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by lsmith13** [comment]
@haileybrink I am happy to work on it today if that works for you! I am finishing the statewide checks and it would be easier to organize the 1983 before going through the statewide.

**Comment by haileybrink** [comment]
Yes that works for me! Thank you!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by lsmith13** [comment]
@regan008 In the 1983 data most of the PO Boxes are listed under notes, as well as some of the descriptions listed, and the descriptions column seems to be focused on phone number info and some are missing what is actually listed in the description from the GG issue. Should I go through and correct this?

**Comment by regan008** [comment]
@lsmith13 For now, lets not correct it unless its egregiously wrong. I'd say if its missing info in the description you could add it back in. I'm less concerned with PO Boxes than I am missing pieces of the description. Some of this is copy and paste issues with the guide and some is probably a misunderstanding about what goes where during transcription

**Comment by heinz-am** [comment]
@lsmith13 How close are you to finishing the 1983 guide in terms of hours and estimated date (for our planning purposes)? Also, please leave a note when it is completed. Thank you!

**Comment by lsmith13** [comment]
@heinz-am I should be finished by Wednesday. I'm thinking about 10 more hours

**Comment by lsmith13** [comment]
@regan008 In the 1983 issue many of the categories in the "type" column are not accurate to the listing but are not totally off base. An example is that the GG issue has the listings under "Special Interest Groups" and the data has the type "groups" selected. 
Is this something I should spend time correcting?

**Comment by regan008** [comment]
@lsmith13 We should probably try to correct these but I wouldn't spend a huge amount of time on it because its not actually a crucial step for us to move forward with the code on our end. Lets add this to a list of fixes that we want to make but that aren't time sensitive and then we can circle back to these once we have a working dataset for the coding portion of the project.

**Comment by lsmith13** [comment]
@regan008 @heinz-am The only step I did not complete was correcting the "type" column, the rest of the list is complete and it should be ready for geocoding.

**Comment by heinz-am** [comment]
Thanks, @lsmith13 ! @regan008, if the "type" issue is recorded for the ongoing list, would it be possible to move to the geocoding column? Just trying to keep my wires straight... ;)
